internal/pkg/manifest/errors.go

```diff
@@ -98,6 +98,30 @@ func (e *errFieldMutualExclusive) Error() string {
 	return fmt.Sprintf(`must specify one, not both, of "%s" and "%s"`, e.firstField, e.secondField)
 }
 
+type errSpecifiedBothIngressFields struct {
+	firstField  string
+	secondField string
+}
+
+func (e *errSpecifiedBothIngressFields) Error() string {
+	return fmt.Sprintf(`must specify one, not both, of "%s" and "%s"`, e.firstField, e.secondField)
+}
+
+// RecommendActions returns recommended actions to be taken after the error.
+func (e *errSpecifiedBothIngressFields) RecommendActions() string {
+	privateOrPublicField := strings.Split(e.firstField, ".")[0]
+	return fmt.Sprintf(`
+It looks like you have specified mutually exclusive fields for Load Balancer configuration.
```

Comment:
I think we can be more specific here. For example, for a customer that has `http.public.security_groups.ingress.cdn` and `http.ingress.source_ips`, we can say something like 

> "It looks like you specified ingress under both `http.public.security_groups.ingress` and `http.public.ingress`. 
> After Copilot v1.X, we have deprecated `http.public.security_groups.ingress` in favor of `http.public.ingress`.  This means that `http.public.security_groups.ingress.cdn`  is removed in favor of `http.public.ingress.cdn`:
> <manifest example>

Just an idea to better communicate the solution to customers so that there is no :panic